.github/workflows/mvn-verify-check.yml

```diff
@@ -95,3 +95,30 @@ jobs:
           -DskipTests
           -Dskip
           -Dmaven.javadoc.skip
+
+  verify-modules-with-jdk11:
+    needs: get-noSnapshot-versions-from-dist
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2 # refs/pull/:prNumber/merge
```

Comment:
please upmerge the base ref, and we should use actions/checkout@v3 to avoid deprecations